modules/library/api/src/main/java/org/geotools/util/Converters.java

```diff
@@ -184,6 +194,32 @@ private static FactoryRegistry getServiceRegistry() {
         }
         return null;
     }
+    
+    public static Converter getConverter(Class source, Class target, final Hints hints) {
```

Comment:
need a comment for a public API method.

Q: should you call this factory method from convert( source, target, hints ).